tchannel/zipkin/formatters.py

```diff
@@ -160,7 +167,9 @@ def thrift_formatter(trace, annotations, isbased64=False):
         host=host,
         parentId=i64_to_string(trace.parent_span_id),
         annotations=thrift_annotations,
-        binaryAnnotations=binary_annotations
+        binaryAnnotations=binary_annotations,
+        debug=False,
```

Comment:
a different issue - we should be carrying over the debug flag from the parent span. But it's not critical.